src/runtime/virtcontainers/pkg/types/pcipath.go

```diff
@@ -0,0 +1,98 @@
+// Copyright Red Hat.
```

Comment:
Suggest, to be consistent:

```suggestion
// Copyright (c) 2021 Red Hat.
```

Reply:
The form I have is what's recommended by Red Hat for open source contributions.